crates/components/deskc-ast/src/span.rs

```diff
@@ -9,10 +9,10 @@ pub struct WithSpan<T> {
     pub value: T,
 }
 
-pub fn dummy_span<T>(value: T) -> WithSpan<T> {
```

Comment:
This function gives a dummy value to make it easier on the test. I regret that it has no comments and `#[cfg(test)]`.